code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -111,6 +114,7 @@
 /obj/item/gun_upgrade/trigger/dangerzone
 	name = "Frozen Star \"Danger Zone\" Trigger"
 	desc = "When you need that extra edge."
+	matter = list(MATERIAL_STEEL = 2, MATERIAL_DIAMOND = 1)
```

Comment:
Why does this cost diamond? could just be silver or maybe gold since technology

Reply:
Done.